src/sql/workbench/services/serverGroup/browser/serverGroupDialog.ts

```diff
@@ -128,8 +128,10 @@ export class ServerGroupDialog extends Modal {
 				this.preventDefaultKeyboardEvent(event);
 				this.focusNext();
 			} else if (event.equals(KeyCode.RightArrow) || event.equals(KeyCode.LeftArrow)) {
-				this.preventDefaultKeyboardEvent(event);
-				this.focusNextColor(event.equals(KeyCode.RightArrow));
+				if (event.target === this._serverGroupContainer) {
```

Comment:
I am not sure why did we add the listener on the body element, i think we should add the listener to the server group container and we can remove the code that handles the tab keys because it is the default behavior.

Reply:
That's a good point. For some reason we have some custom logic in the focusNext/focusPrevious functions though, but in general I'm not sure what they would be doing that shouldn't be the default behavior for tabbing.

So I'd suggest looking into that instead to make sure we're doing the right thing there and hopefully being able to clean it up a bit.